src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+public class Formatter {
```

Comment:
Perhaps more formatting-related stuff (e.g. `DIVIDER`) can be moved to the `Formatter`? May be even things like this should be moved to the Formatter? Note that it may not always be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.